### PR TITLE
Add in a prodcheck script

### PR DESCRIPTION
**Pull request by paultag**

### CODE REVIEW

**Comment by paultag** [comment]
Small helper script to check if prod is in sync with master

**Comment by paultag** [comment]
```sh
[paultag@va:~/dev/usds/caseflow/scripts][11:38 AM] ♥  ./prodcheck 
commit d7b36a02ffde1d0f050f4919eb75f5e9cb40cb88
Author: Paul Tagliamonte <paul.tagliamonte@va.gov>
Date:   Thu Feb 25 11:37:02 2016 -0500

    Add in a prodcheck script
[paultag@va:~/dev/usds/caseflow/scripts][11:38 AM] ♥  git reset origin/master
[paultag@va:~/dev/usds/caseflow/scripts][11:38 AM] ♥  ./prodcheck 
Up to date!
[paultag@va:~/dev/usds/caseflow/scripts][11:38 AM] ♥  
```

File: scripts/prodcheck [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    SITE="caseflow"
+fi
+
+URI="${SITE}.ds.va.gov"
+
+if [ "x${ENV}" != "x" ]; then
+    URI="${ENV}.${URI}"
+fi
+
+HASH=$(curl --insecure https://caseflow.ds.va.gov/caseflow.build 2>/dev/null)
```
Comment (paultag) [inline scripts/prodcheck]: this makes me cry inside, but the VA CA is super not in the public trust stores

File: scripts/prodcheck [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+# prodcheck caseflow preprod
+
+if [ "x${SITE}" = "x" ]; then
+    SITE="caseflow"
+fi
+
+URI="${SITE}.ds.va.gov"
+
+if [ "x${ENV}" != "x" ]; then
+    URI="${ENV}.${URI}"
```
Comment (shanear) [inline scripts/prodcheck]: what is this URI variable being used for?

File: scripts/prodcheck [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+# prodcheck caseflow preprod
+
+if [ "x${SITE}" = "x" ]; then
+    SITE="caseflow"
+fi
+
+URI="${SITE}.ds.va.gov"
+
+if [ "x${ENV}" != "x" ]; then
```
Comment (shanear) [inline scripts/prodcheck]: is this how you check for blank variables in shell? lol shell.

**Pull request closed without merging**